File: exercises/03_exercise_conditional_statements_advanced/09_ski_trip.py
days = int(input())
type_of_room = input()
rating = input()
night = days - 1
price_per_night = 0
if type_of_room == "room for one person":
    price_per_night = 18
elif type_of_room == "apartment":
    price_per_night = 25
    if days < 10:
        price_per_night *= 0.7
    elif days >= 10 and days <= 15:
        price_per_night *= 0.65
    elif days > 15:
        price_per_night *= 0.5
elif type_of_room == "president apartment":
    price_per_night = 35
    if days < 10:
        price_per_night *= 0.9
    elif days >= 10 and days <= 15:
        price_per_night *= 0.85
    elif days > 15:
        price_per_night *= 0.8
total = price_per_night * night
if rating == "positive":
    total *= 1.25
elif rating == "negative":
    total *= 0.9
print(f"{total:.2f}")

File: 05_JavaScriptBasics/03_MoreExercises/03_conditionalStatementsAdvanced/09_skiTrip.js
function skiTrip(input){
    let numDays = Number(input[0]);
    let typeRoom = input[1];
    let rating = input[2];
    let numNights = numDays - 1

    let roomPrice = 18;
    let apartmentPrice = 25;
    let presidetApartment = 35;
    let totalPrice = 0

    if (numDays < 10){
        switch (typeRoom){
            case "room for one person":
                totalPrice = numNights * roomPrice;
                break;
            case "apartment":
                totalPrice = (numNights * apartmentPrice) * 0.70;
                break;
            case "president apartment":
                totalPrice = (numNights * presidetApartment) * 0.90;
                break;
        }
    } else if (numDays <= 15){
        switch (typeRoom){
            case "room for one person":
                totalPrice = numNights * roomPrice;
                break;
            case "apartment":
                totalPrice = (numNights * apartmentPrice) * 0.65;
                break;
            case "president apartment":
                totalPrice = (numNights * presidetApartment) * 0.85;
                break;
        }
    } else {
        switch (typeRoom){
            case "room for one person":
                totalPrice = numNights * roomPrice;
                break;
            case "apartment":
                totalPrice = (numNights * apartmentPrice) * 0.50;
                break;
            case "president apartment":
                totalPrice = (numNights * presidetApartment) * 0.80;
                break;
        }
    }
    if (rating === "positive"){
        totalPrice *= 1.25
    } else {
        totalPrice *= 0.90
    }
    console.log(`${totalPrice.toFixed(2)}`)
}

skiTrip(["14", "apartment", "positive"]);
skiTrip(["30", "president apartment", "negative"]);
skiTrip(["12", "room for one person", "positive"]);
skiTrip(["2", "apartment", "positive"]);
